### PR TITLE
Use custom fee

### PR DESCRIPTION
**Pull request by garyghayrat**

### CODE REVIEW

**Review by apbendi — APPROVED** [review]
Looks good to me @garyghayrat. Thanks!

File: src/CreatorToken.sol [hunk 1]
```diff
@@ -65,11 +65,15 @@ contract CreatorToken is ERC721 {
     string memory _name,
     string memory _symbol,
     address _creator,
+    uint256 _creatorFee,
     address _admin,
+    uint256 _adminFee,
     IERC20 _payToken
   ) ERC721(_name, _symbol) isNotAddressZero(_creator) isNotAddressZero(_admin) {
     creator = _creator;
+    CREATOR_FEE_BIPS = _creatorFee;
```
Comment (apbendi) [inline src/CreatorToken.sol:74]: Thinking about it now, we should probably have some bounds around the max fee that you can set. Certainly the total has to be less than 1,000 or things break. Realistically there's probably also some cap we could enforce on each fee, maybe 25%.

File: test/CreatorToken.t.sol [hunk 2]
```diff
@@ -17,6 +17,8 @@ contract CreatorTokenTest is Test {
   string PAY_TOKEN_NAME = "Payment Token";
   string PAY_TOKEN_SYMBOL = "PAY";
   uint256 BASE_PAY_AMOUNT = 1e18; // Because our test token has 18 decimals
+  uint256 constant CREATOR_FEE = 700; // 7%
+  uint256 constant ADMIN_FEE = 300; // 3%
```
Comment (apbendi) [inline test/CreatorToken.t.sol:21]: Let's pull out an issue to think about how to run the tests with varying fee values.

Comment (garyghayrat) [reply test/CreatorToken.t.sol:23]: Created this issue: https://github.com/showtime-xyz/creator-tokens/issues/20

**Pull request merged**